app/Models/Build.php

```diff
@@ -124,4 +124,12 @@ protected function serializeDate(DateTimeInterface $date): string
     {
         return $date->format('Y-m-d H:i:s');
     }
+
+    /**
+     * @param $value
+     */
+    public function setForcedAttribute($value): void
+    {
+        $this->attributes['forced'] = filter_var($value, FILTER_VALIDATE_BOOLEAN);
```

Comment:
I didn't know `on` means `true`, nice!

Reply:
I hate html checkboxes so much :angry: 
This should help parse `true/false`, `"true"/"false"`, `1/0`, `on/off`...

Reply:
Just discovered this https://laravel.com/docs/8.x/validation#rule-accepted 💥